airo-typing/airo_typing/__init__.py

```diff
@@ -69,11 +69,67 @@
 shorthand notation is ^C T^B_A, where C is the frame the velocity is measured in, B is the frame the velocity is expressed in.
 """
 
-# manipulator types
+#####################
+# Manipulator types #
+#####################
 
 JointConfigurationType = np.ndarray
 """an (N,) numpy array that represents the joint angles for a robot"""
 
+#########################
+# Single-arm path types #
+#########################
+JointPathType = List[JointConfigurationType]
```

Comment:
I'm not convinced of the need for a separate List and array type? Can't we just always use numpy arrays or always use Lists to avoid confusion?